pyquil/job_results.py

```diff
@@ -100,7 +100,7 @@ def recover_complexes(coef_string, classical_addresses):
     """
     num_octets = len(coef_string)
     num_addresses = len(classical_addresses)
-    num_memory_octets = _round_to_next_multiple(num_addresses, 8) / 8
+    num_memory_octets = _round_to_next_multiple(num_addresses, 8) // 8
```

Comment:
Hmmm... Mypy has been used for a while in Zulip. This is the best reference point to see how it is used in production that I know of. You can see how it can be included in Python 2 via comments, e.g. https://github.com/zulip/zulip/blob/5c9f02924fbb7b4ae628ccd1dc0ebd747035ef73/zerver/lib/actions.py#L447

I will fortify the repo with Python 3 check soonish, brb.